### PR TITLE
fix: fonts it's setup for desktop and conversely

### PR DESCRIPTION
**Pull request by sonya0504**
### Scope of work

- Added `reverse` function to set correct order for fonts properties

### CODE REVIEW

**Comment by pspaczek** [comment]
@sonya0504 please use your fork next time

File: scripts/figma-to-css.mjs [hunk 1]
```diff
@@ -97,6 +97,7 @@ const figmaToCss = async () => {
         case 'Frame 1105':
           if (frame?.children.length > 1) {
             return frame.children
+              .reverse()
               ?.reduce((acc, { children }) => acc.concat(children), [])
               ?.filter(({ characters }) => !characters?.match(/Desktop|Mobile/g))
               ?.map(getFontProperties);
```
Comment (pspaczek) [inline scripts/figma-to-css.mjs:103]:
```suggestion
              ?.map(getFontProperties)
              ?.reverse();
```
WDYT? IMO .reverse at last operation looks better (for me) 
WHY? When I get results without .reverse I have "right" result but I need to reverse it to adjust Desktop -> Mobile order to Mobile->Destkop

File: scripts/figma-to-css.mjs [hunk 2]
```diff
@@ -97,6 +97,7 @@ const figmaToCss = async () => {
         case 'Frame 1105':
           if (frame?.children.length > 1) {
             return frame.children
+              .reverse()
```
[the remark this answers is not in the document]
Comment (sonya0504) [reply scripts/figma-to-css.mjs:100]: Yes sure. Thank you

File: scripts/figma-to-css.mjs (hunk 1 above)
Comment (sonya0504) [reply scripts/figma-to-css.mjs]: changed

**Review by pspaczek — APPROVED** [review]
LGTM! Thaks @sonya0504

**Pull request merged**